solarized.el

```diff
@@ -126,68 +131,129 @@ Related discussion: https://github.com/bbatsov/solarized-emacs/issues/158"
+      (yellow    . ,yellow)
+      (orange    . ,orange)
+      (red       . ,red)
+      (magenta   . ,magenta)
+      (violet    . ,violet)
+      (blue      . ,blue)
+      (cyan      . ,cyan)
+      (green     . ,green)
+
+      ;; Darker and lighter accented colors
+      ;; Only use these in exceptional circumstances!
+      (yellow-d  . ,(solarized-color-blend yellow  darkest-base   0.80 2))
+      (yellow-l  . ,(solarized-color-blend yellow  brightest-base 0.80 2))
+      (orange-d  . ,(solarized-color-blend orange  darkest-base   0.80 2))
+      (orange-l  . ,(solarized-color-blend orange  brightest-base 0.80 2))
+      (red-d     . ,(solarized-color-blend red     darkest-base   0.80 2))
+      (red-l     . ,(solarized-color-blend red     brightest-base 0.80 2))
+      (magenta-d . ,(solarized-color-blend magenta darkest-base   0.80 2))
+      (magenta-l . ,(solarized-color-blend magenta brightest-base 0.80 2))
+      (violet-d  . ,(solarized-color-blend violet  darkest-base   0.80 2))
+      (violet-l  . ,(solarized-color-blend violet  brightest-base 0.80 2))
+      (blue-d    . ,(solarized-color-blend blue    darkest-base   0.80 2))
+      (blue-l    . ,(solarized-color-blend blue    brightest-base 0.80 2))
+      (cyan-d    . ,(solarized-color-blend cyan    darkest-base   0.80 2))
+      (cyan-l    . ,(solarized-color-blend cyan    brightest-base 0.80 2))
+      (green-d   . ,(solarized-color-blend green   darkest-base   0.80 2))
+      (green-l   . ,(solarized-color-blend green   brightest-base 0.80 2)))))
+
+(defvar solarized-color-palette-alist
+  '((s-base03  . "#002b36")
+    (s-base02  . "#073642")
+    ;; emphasized content
+    (s-base01  . "#586e75")
+    ;; primary content
+    (s-base00  . "#657b83")
+    (s-base0   . "#839496")
+    ;; comments
+    (s-base1   . "#93a1a1")
+    ;; background highlight light
+    (s-base2   . "#eee8d5")
+    ;; background light
+    (s-base3   . "#fdf6e3")
+
+    ;; Solarized accented colors
+    (yellow    . "#b58900")
+    (orange    . "#cb4b16")
+    (red       . "#dc322f")
+    (magenta   . "#d33682")
+    (violet    . "#6c71c4")
+    (blue      . "#268bd2")
+    (cyan      . "#2aa198")
+    (green     . "#859900")
+
+    ;; Darker and lighter accented colors
+    ;; Only use these in exceptional circumstances!
+    (yellow-d  . "#7B6000")
+    (yellow-l  . "#DEB542")
+    (orange-d  . "#8B2C02")
+    (orange-l  . "#F2804F")
+    (red-d     . "#990A1B")
+    (red-l     . "#FF6E64")
+    (magenta-d . "#93115C")
+    (magenta-l . "#F771AC")
+    (violet-d  . "#3F4D91")
+    (violet-l  . "#9EA0E5")
+    (blue-d    . "#00629D")
+    (blue-l    . "#69B7F0")
+    (cyan-d    . "#00736F")
+    (cyan-l    . "#69CABF")
+    (green-d   . "#546E00")
+    (green-l   . "#B4C342"))
+  "The solarized color palette alist.")
 
 ;;; Setup Start
-(defmacro solarized-with-color-variables (variant &rest body)
+(defmacro solarized-with-color-variables (variant color-palette &rest body)
```

Comment:
@thomasf @bbatsov @conao3 Adding this extra argument has broken the signature of this macro and will break user code that uses it.  For example, I use this macro extensively in my own configuration.

Reply:
We have made a stable release so there are no promises broken *yet*.  Sorry for not noticing this discssion, github has decided to stop notifying me properly. I need to go to bed now, will try to read up on these issues again tomorrow.